sdk/python/kfp_tekton/compiler/compiler.py

```diff
@@ -129,6 +129,7 @@ def _create_pipeline_workflow(self, args, pipeline, op_transformers=None, pipeli
         task['retries'] = op.num_retries
 
     # add timeout params to task_refs, instead of task.
+    pipeline_conf = pipeline.conf
```

Comment:
this seems unneccessary, we can just use `pipeline.conf` later on

Reply:
I think `pipeline` get overwritten in the next few lines when defining the tekton pipeline. My pr https://github.com/kubeflow/kfp-tekton/pull/65 changes the tekton pipeline variable so it won't conflict with the original pipeline object.